lib/sdb/right_sdb_interface.rb

```diff
@@ -637,7 +637,8 @@ def select(select_expression, next_token = nil)
         link   = generate_request("Select", request_params)
         result = select_response_to_ruby(request_info( link, QSdbSelectParser.new ))
       end while true
-    rescue Exception
+    rescue Exception => e
+      Rails.logger.info(e);
```

Comment:
Do you want to add some header text so we can easily find this in the log e.g. `Rails.logger.error("EXCEPTION in right_sdb_interface.rb: #{e}")`